src/frontend/gatsby/src/components/SearchInput/SearchInput.jsx

```diff
@@ -22,6 +22,7 @@ const useStyles = makeStyles((theme) => ({
     height: '55px',
     backgroundColor: theme.palette.background.default,
     paddingLeft: '10px',
+    paddingRight: '65px',
```

Comment:
This works really well, great job.

Let's tweak the value here slightly so the space to the right of the search button, and what's now on the left of it, are the same.  The left-hand side is too large by a bit.  Maybe 60px?  Just guessing.

<img width="74" alt="Screen Shot 2021-01-06 at 7 26 14 PM" src="https://user-images.githubusercontent.com/427398/103835792-2ea5eb80-5055-11eb-85fd-6f25950b0c56.png">

Reply:
Ok, I will go fix it! :)